cmake/xpfunmac.cmake

```diff
@@ -2144,14 +2144,15 @@ function(xpAddCoverage)
   endif()
   if(P_CSharp)
     add_custom_target(precoveragecsharp
-      COMMAND ${CMAKE_COMMAND} -E make_directory coveragecsharp
+      COMMAND dotnet tool install dotnet-reportgenerator-globaltool --version 5.1.26 --global || (exit 0) 
```

Comment:
I added back in this command to install it. I added (exit 0) on the end because before if I ran the precoveragecsharp twice, the target would fail the second time because it is already installed. The command will still fail, but the target will always pass

Reply:
I also changed the order, delete the directory, then add it back